.github/workflows/py-run-tests.yml

```diff
@@ -7,6 +7,8 @@ on:
     paths:
       - 'python/**'
   pull_request:
+    paths:
+      - python/**
```

Comment:
Just need to trigger this workflow when change something in Python folder. Make sense?